.github/workflows/config-template.yaml.j2

```diff
@@ -50,13 +55,39 @@ hub:
       scope:
         - read:user
         - read:org
-    JupyterHub:
-      authenticator_class: github
+  {%- elif authenticator_class == 'dummy' %}
+    DummyAuthenticator:
+      password: "{{ auth_password }}"
+  {%- endif %}
+{%- endif %}
+
   db:
     pvc:
-      storageClassName: gp2
+      storageClassName: {{ 'do-block-storage' if RELEASE == 'aiidalab-marketplace' else 'gp2' }}
+
   extraConfig:
     myConfig: |
+{%- if RELEASE == 'aiidalab-marketplace' %} 
+      # The code has no indent, in order to keep the indent as in this template
+      # wrap the include in a macro name it mpoauth(), the pass this macro function 
+      # through the indent filter.
+      {% macro mpoauth() %}{% include "./marketplace/oauthenticator.py" %}{% endmacro %}
+      {{ mpoauth()|indent(6) }}
```

Comment:
Thank you for the explanation. Did you come up with this your self or was this advised by a SO comment or so? If yes, maybe we can link to that?